app/presenters/tree_builder_vandt.rb

```diff
@@ -16,7 +16,10 @@ def root_options
 
   def x_get_tree_roots(count_only, options)
     objects = count_only_or_objects_filtered(count_only, EmsInfra, "name", :match_via_descendants => VmOrTemplate)
-    objects.collect! { |o| TreeBuilderVmsAndTemplates.new(o, options.dup).tree } unless count_only
+    unless count_only
+      show_vms = Settings.server.fetch(:display_vms) { true }
```

Comment:
Why fetch? You should be able to just call it as long as it exists in the settings.yml (which it should).